.circleci/config.yml

```diff
@@ -404,6 +416,106 @@ references:
     - Demisto SDK Nightly - Run against Cortex XSOAR:
         context: sdk_nightly_env
 
+  bucket_upload_jobs: &bucket_upload_jobs
+    - Setup Environment
+    - Create Instances:
+        context: bucket_upload_env
+        requires:
+          - Setup Environment
+    - Run Unit Testing And Lint:
+        context: bucket_upload_env
+        requires:
+          - Setup Environment
+    - Run Validations:
+        context: bucket_upload_env
+        requires:
+          - Setup Environment
+    - Install Packs in Demisto 6_0:
+        requires:
+          - Create Instances
+    - Install Packs in Demisto Marketplace - Master:
+        requires:
+          - Create Instances
+    - Upload Packs:
+        context: bucket_upload_env
+        requires:
+          - Run Unit Testing And Lint
+          - Run Validations
+          - Install Packs in Demisto 6_0
+          - Install Packs in Demisto Marketplace - Master
+
+
+  upload_packs_to_marketplace_storage: &upload_packs_to_marketplace_storage
+    run:
+      name: Upload Packs To Marketplace Storage
+      command: |
+        # IMPORTANT! By removing this condition, branch changes will be uploaded to production bucket.
```

Comment:
Use the `unless` syntax so that the job will not be executed at all if it's not branch master.

See here https://github.com/demisto/content/blob/b6ccdd82cfad7a6e81b675e5aa8e0e1f4772396a/.circleci/config.yml#L615 for example

Reply:
not sure i understood where you want me to use this syntax